draft-ietf-dnsop-svcb-https.md

```diff
@@ -867,7 +867,30 @@ compliant recursive resolvers.  When SvcDomainName is ".", server operators
 SHOULD NOT include these hints, because they are unlikely to convey any
 performance benefit.
 
+## "mandatory" {#mandatory}
 
+In a ServiceForm RR, a SvcParamKey is considered "essential" if the RR will not
+function correctly for clients that ignore this SvcParamKey.  Each SVCB
+protocol mapping SHOULD specify a default set of essential keys.  The
+SvcParamKey "mandatory" is used to indicate the set of essential keys for this
+RR (overriding the default set).
+
+A ServiceForm RR is considered "compatible" with a client if the client
+implements support for all its essential keys.
+
```

Comment:
I'd add a note here that if no ServiceForm RRs in the RRset are compatible then the client must treat the RRset as empty.

Reply:
I think that is already mandated by each section on client behavior, but I added a non-normative note to this effect.